### DISCUSSION TITLE
Not able to disable / make it read only for the radio field

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#66] [Q&A] [answered]
Hi @manojadams 

We are trying to disable the gender radio field in this form but having trouble doing it.
We tried giving the fields isDisabled and isReadOnly for the gender field but couldn't do it.

[Here is the code example.](https://codesandbox.io/p/sandbox/cool-flower-zlzm36?file=%2Fsrc%2FCustomerFeedbackForm.js%3A94%2C13)

**Comment by manojadams** [comment, marked as answer]
@venkateshpullaganti Disabled has been fixed now in the latest version: **mui-forms@2.0.3**

Also readonly is not supported for radio fields:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#readonly

**Reply by venkateshpullaganti** [reply]
Okay @manojadams Got it. Thank you
